site/docs/README.md

```diff
@@ -43,7 +43,7 @@ import { Bot } from "grammy";
 
 const bot = new Bot(""); // <-- put your bot token here (https://t.me/BotFather)
 
-// Reply to any message with "Hi there!".
+// Reply to any message with “Hi there!”.
```

Comment:
I'm not sure we should use non-ASCII256 characters in code. Maybe I have pre-JS mentality here, did we already do this in any other places before?

I do realise that the Chinese translation has Chinese characters in the comments, and I believe that this is the right thing to do—what's your opinion on the matter?

Reply:
I've already changed the comments too. I forgot this one.

Reply:
Let me fix the comments too.